Homework_11/src/test/java/test/java/OnfootTest.java

```diff
@@ -0,0 +1,32 @@
+package test.java;
+
+import org.junit.Before;
+import org.junit.Test;
+import tat.maven.Onfoot;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by Nick on 30/10/16.
+ */
+public class OnfootTest {
+    Onfoot onfoot;
+
+    @Before
+    public void setUp() {
+        Onfoot onfoot = new Onfoot(1.0);
+    }
+
+    @Test
+    public void getTime() throws Exception {
+        assertTrue(onfoot.getTime() == 0);
```

Comment:
1. This test checks nothing (only that trivial getter works well, you have not to test it)
2.  At any case, use assertEquals instead of assertTrue